src/main/java/www/purple/mixxy/views/layout/header.ftl.html

```diff
@@ -1,15 +1,47 @@
 <header>
-	<a href="/"><img src="/assets/images/logo.png" alt="Mixxy - Make Art Great!" width=200 height=auto></a>
-	Make Art Great!
-	<ul class="navbar-right nav navbar-nav">
-		<#if (session.username)??>
-			<li><a href="#">${session.username}</a></li>
-			<li><a href="/logout">${i18n("header.logout")}</a></li>
-			<#else>
-				<li><a href="/login?provider=facebook" id="facebook">${i18n("header.facebook")}</a></li>
-				<li><a href="/login?provider=google" id="google">${i18n("header.google")}</a></li>
-				<li><a href="/login?provider=deviantart" id="deviantart">${i18n("header.deviantart")}</a></li>
-		</#if>
-
-	</ul>
+    <div class="ui container">
+        <div class="ui centered padded stackable grid">
+            <div class="row">
+                <div class="six wide left floated column">
+                    <div class="row">
+                        <a href="/" class="ui medium image"><img src="/assets/images/logo.png"
+                                                                 alt="Mixxy - Make Art Great!"></a>
+                    </div>
+                    <div class="row">Make Art Great!
+                    </div>
+                </div>
+                <div class="eight wide right floated column">
+                    <div class="row">
+                        <div class="ui text menu">
+                            <div class="ui right dropdown item">
+                                <#if (session.username)??>
+                                    <p>Hi, <a class="item" href="#">${session.username}</a>
+                                        <i class="dropdown icon"></i></p>
+                                    <div class="menu">
+                                        <a class="item">Subscriptions</a>
+                                        <a class="item">Profile</a>
+                                        <a class="item" href="/logout">${i18n("header.logout")}</a>
+                                    </div>
+                                    <#else>
+                                        <p>Sign In
+                                            <i class="dropdown icon"></i></p>
+                                        <div class="menu">
+                                            <a class="item" href="/login?provider=facebook" id="facebook">${i18n("header.facebook")}</a>
+                                            <a class="item" href="/login?provider=google" id="google">${i18n("header.google")}</a>
+                                            <a class="item" href="/login?provider=deviantart" id="deviantart">${i18n("header.deviantart")}</a>
```

Comment:
The header.fb,google,deviantart are not the same id's at mentioned in the java/conf/message.properties file